applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -3,5 +3,182 @@
++----------------------------------+------------------------+                                  +                      +------------------------------------+
+|                                  |                        |                                  |                      | ``leds``                           |
++----------------------------------+------------------------+                                  +                      +------------------------------------+
+|                                  |                        |                                  |                      | ``board``                          |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``nrf_desktop_main``             | ``module_state_event`` |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``ble_state``                    | ``ble_peer_event``     |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``usb_state``                    | ``usb_state_event``    |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``hid_state``                    | ``hid_report_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``power_manager``                | ``power_down_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+|                                  | ``wake_up_event``      |                                  |                      |                                    |
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+
+Configuration
+*************
+
+This module uses Zephyr's :ref:`zephyr:power_management` subsystem.
+It depends on :option:`CONFIG_HAS_SYS_POWER_STATE_DEEP_SLEEP_1` being enabled and selects :option:`CONFIG_DEVICE_POWER_MANAGEMENT` and :option:`CONFIG_SYS_POWER_DEEP_SLEEP_STATES`.
```

Comment:
The dependencies are related to `CONFIG_DESKTOP_POWER_MANAGER_ENABLE`. We should move the next line here I think.

Reply:
I have put it at the first line now.